### ISSUE TITLE
Antimicro Fork?

### ISSUE THREAD

**Issue by trymeouteh** [juliagoda/antimicro#58]
This is a fork of Antimicro but is this now the official version? Will development continue for this application here?

I don't want Antimicro to die and I would like to see someone or a group to dedicate some time to ensure Antimicro stays up to date with a release or two a year. Anitmicro is the best open source windows & linux controller mapping software out there.

**Comment by juliagoda** [comment]
Yes. On the one hand it's fork (with whole history of commits/contributors included in it - mirrored) and on the other hand it's other official site. It's such complicated because no one wanted to reply to my email from AntiMicro about takeover. So after some time as first I created fork and I wrote about continuation on this site. I wrote about it in one of issues from AntiMicro project. Sadly, someone from old team removed the issue and only this one, just because I told about my project (Subject of the issue was about continuation anyway and opened by someone else). After that I was never trying to contact the old team, because It didn't and doesn't make any sense. They haven't been time to answer users questions for two years but there was enough time to delete a thread without any response.

I try to do my best and improve anything here, whenever I have time.
